pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -176,12 +176,17 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	// docker-in-docker (dind) deployments are used for testing
 	// networking plugins.  Running openshift under dind won't work
 	// with the real oom adjuster due to the state of the cgroups path
-	// in a dind container that uses systemd for init.
+	// in a dind container that uses systemd for init.  Similarly,
+	// cgroup manipulation of the nested docker daemon doesn't work
+	// properly under centos/rhel and should be disabled by setting
+	// the name of the container to an empty string.
 	//
-	// TODO(marun) Make dind cgroups compatible with openshift
+	// This workaround should become unnecessary once user namespaces
 	if value := cmdutil.Env("OPENSHIFT_DIND", ""); value == "true" {
```

Comment:
I don't think this should be backed out without an alternate solution, since it is required for networking ci to prevent regressions.  What alternative approach to disabling cgroup the kubelet's broken cgroup manipulation would you suggest?